src/fittable_models.jl

```diff
@@ -0,0 +1,45 @@
+# Convenience function for wavelength conversion
+aa_to_invum(wave::Real) = 10000 / wave
+aa_to_invum(wave::Quantity) = aa_to_invum(ustrip(u"angstrom", wave))
+
+"""
+    FM90(C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)(λ::Real)
+
+The parameters C1, C2, C3, C4, xo and γ are y-intercept, slope, bump amplitude, FUV rise amplitude,
+bump centroid and bump width respectively.
+
+Fitzpatrick & Massa (1990) 6 parameter ultraviolet shape model, this model is only applicable at UV wavelengths.
+
+If `λ` is a `Unitful.Quantity` it will be automatically converted to Å and the
+returned value will be `UnitfulAstro.mag`.
+"""
+struct FM90
+    C1::Real
+    C2::Real
+    C3::Real
+    C4::Real
+    xo::Real
+    γ::Real
+end
+
+FM90() = FM90(0.10, 0.70, 3.23, 0.41, 4.60, 0.9)
+
+function (z::FM90)(λ::Real)
```

Comment:
Does this look good for the interface? @giordano @mileslucas

Reply:
Yeah, I think this is good.

Reply:
I wanted to support `Quantity` as a form of input for lamda. I tried this:
```
function (z::FM90)(λ::Quantity) = (z)(ustrip(u"angstrom", λ)) * u"mag"
```
but it says `syntax: unexpected "="`. How to work this out?

Reply:
Remove the `function` keyword :wink:

Reply:
Would the function offer better performance if written as `function (z::FM90)(x::T) where T <: Number`?

Reply:
Nope. Typing is for dispatch only, it won’t make your code faster. Also, in general, if you want to ask “is it better performance” is really hard to judge from visual inspection only. Using BenchmarkTools.jl and playing around yourself will be far more useful.

Reply:
To add to what Miles said, you may be interested in reading https://white.ucc.asn.au/2020/04/19/Julia-Antipatterns.html

Reply:
Thank you so much for all the help!